src/interface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+interface IShapeDrawRectangle
+{
+  public function checkSidesValue($height, $width);
+  public function draw();
+
+  public function getValueSide();
+  public function setValueSide($height, $width);
+}
+
+interface IShapeDrawSquare
+{
+  public function checkSidesValue($length);
+  public function draw();
+
+  public function getValueSide();
+  public function setValueSide($length);
+}
+
+interface IShapeSetColor
```

Comment:
I could be renamed to IColorable or IColorableBlock or IColorableShape

Reply:
Ok, I chose IColorable